core/Tracker/Db/Mysqli.php

```diff
@@ -148,6 +148,39 @@ public function disconnect()
         $this->connection = null;
     }
 
+    /**
+     * @param \mysqli_stmt $stmt
+     * @param $fields
+     * @return array|bool|false
+     */
+    private function fetchResult($stmt, $fields) {
+
+        $values = array_fill(0, count($fields), null);
+
+        $refs = array();
+        foreach ($values as $i => &$f) {
+            $refs[$i] = &$f;
+        }
```

Comment:
Is `$refs` needed? It doesn't seem to be used anywhere.

Reply:
It was done like this in Zend I suppose it turns it into a reference.

Reply:
that's weird, but ok zend